Update LICENSE.md (#14)
TeamBasedInquiryLearning/library: LICENSE.md

# Copyright and Licensing

All contents of this repository are copyright 2017-2024 Steven Clontz and Drew Lewis.
and are licensed for free public use under the following licenses.

> [Contributions](https://github.com/TeamBasedInquiryLearning/library/blob/main/CONTRIBUTING.md) are warmly welcomed and encouraged.

Reviewer: ```suggestion
Contributions are warmly welcomed and encouraged; see [CONTRIBUTING.md](CONTRIBUTING.md) for details.
```

Reviewer: @siwelwerd consider this further tweak 👆

Reviewer: oh you did, weird, it didn't close this. ah well

> All contents of this repository are copyright 2017-2024 Steven Clontz and Drew Lewis. By contributing to this repository, you agree that you are providing the copyright owners an irrevocable, perpetual, non-exclusive, fully-paid, worldwide license, with the right to sublicense through multiple tiers, to use, distribute, reproduce, modify, adapt, publish, translate, publicly perform, and publicly display or otherwise use your contributions (in whole or in part) in any format or medium now known or later developed, without compensation or notification to or permission from the user of any kind.
> 
> All contents of this repository are licensed for free public use under the following licenses:

Reviewer: ```suggestion
All contents of this repository are copyright 2017-2024 Steven Clontz and Drew Lewis.
and are licensed for free public use under the following licenses.
[Contributions](https://github.com/TeamBasedInquiryLearning/library/blob/main/CONTRIBUTING.md) are warmly welcomed and encouraged.
```
